.travis.yml

```diff
@@ -25,4 +25,6 @@ script:
   - if [ "$TRAVIS_OTP_RELEASE" != "19.2" ]; then echo  "dialyzer disabled on OTP < 19.1"; else rebar3 dialyzer; fi
   - rebar3 xref
   - rebar3 eunit
+  - rebar3 eunit
```

Comment:
Intentional for now.  The first run of eunit doesn't seem to pick up Alpaca's generated tests, even if the compile phase has been explicitly done prior.